File: Server/src/main/java/com/github/dr/rwserver/plugin/annotation/EventCentral.java
package com.github.dr.rwserver.plugin.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Target({ElementType.METHOD}) //注解应用类型(应用到方法的注解,还有类的可以自己试试)
@Retention(RetentionPolicy.RUNTIME) // 注解的类型
public @interface EventCentral {
    //  TODO 暂无实现
}

File: Server/src/main/java/com/github/dr/rwserver/plugin/annotation/EventCentral.kt
/*
 * Copyright 2020-2022 RW-HPS Team and contributors.
 *
 * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
 * Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
 *
 * https://github.com/RW-HPS/RW-HPS/blob/master/LICENSE
 */

package com.github.dr.rwserver.plugin.annotation

@Target(
    AnnotationTarget.FUNCTION,
    AnnotationTarget.PROPERTY_GETTER,
    AnnotationTarget.PROPERTY_SETTER
) //注解应用类型(应用到方法的注解,还有类的可以自己试试)
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME) // 注解的类型
annotation class EventCentral { //  TODO 暂无实现
}